packages/components/field-label/src/field-label.js

```diff
@@ -96,7 +101,28 @@ FieldLabel.propTypes = {
   hasRequiredIndicator: PropTypes.bool,
   htmlFor: PropTypes.string,
   id: PropTypes.string,
-  horizontalConstraint: PropTypes.oneOf(['s', 'm', 'l', 'xl', 'scale']),
+  horizontalConstraint: PropTypes.oneOf([
+    's',
+    'm',
+    'l',
+    'xl',
+    3,
```

Comment:
I suppose we don't allow sizes below 3, right?

Reply:
Yeah, I kept the same restrictions as before for the smallest constraints allowed for all of the components.

In this case, it was `s`, so the equivalent is `3`.